src/packet_handlers/gameplay_router.rs

```diff
@@ -49,6 +51,14 @@ pub fn route_packet<P: PlayerState, PA: PatchworkState>(
         Packet::ChatMessage(_) => {
             chat_message_router::route_packet(p, conn_id, patchwork_state);
         }
+        Packet::PlayerDigging(block_packet) => {
+            println!("PlayerDigging from gameplay_router");
+            block_state.break_block_serverbound(conn_id, block_packet);
+        }
+        Packet::BlockChange(block_packet) => {
```

Comment:
How would we receive this here? You say it's a clientbound packet but here we are receiving it on the server